src/SourceBuild/content/repo-projects/aspnetcore.proj

```diff
@@ -33,6 +33,8 @@
 
     <!-- Tests are failing to build: https://github.com/dotnet/aspnetcore/issues/60095 -->
     <DotNetBuildTestsOptOut>true</DotNetBuildTestsOptOut>
+
+    <RepoHasRidSpecificArtifacts>true</RepoHasRidSpecificArtifacts>
```

Comment:
I'm worried about the maintenance of this information. Any ideas how to keep this in sync with the repos? I think a repository should be able to start producing RID specific artifacts without a change in the VMR orchestrator.

cc @mmitche for opinions

Reply:
I think that would be better, yes. We usually catch missing artifacts in consuming repos (inside and outside of the VMR) via dependency flow immediately anyway.

I.e. this switch wouldn't have guarded against the recent artifacts publishing regression in runtime.